libnavigator/src/main/java/com/mapbox/navigation/navigator/internal/NavigatorMapper.kt

```diff
@@ -55,7 +55,7 @@ private val SUPPORTED_ROUTE_ALERTS = arrayOf(
     RouteAlertType.INCIDENT
 )
 
-internal class NavigatorMapper {
+class NavigatorMapper internal constructor() {
```

Comment:
Why this change? Developers or other modules don't need to interact with this class, right?

Reply:
no, it is used in `core` module
`navigatorMapper.getIncidentInfo(it.incident)`